options/locale/locale_en-US.ini

```diff
@@ -1281,7 +1246,6 @@ editor.file_delete_success = File "%s" has been deleted.
 editor.name_your_file = Name your file…
 editor.filename_help = Add a directory by typing its name followed by a slash ('/'). Remove a directory by typing backspace at the beginning of the input field.
 editor.or = or
-editor.cancel_lower = Cancel
```

Comment:
How did a `lower` ever end up in the translations?
And why isn't it even lowercased?